notebooks/concepts/Specviz from splot concept notebook.ipynb

```diff
@@ -0,0 +1,3636 @@
+    "        spectrum  are  done  in wavelength space and the second spectrum\n",
+    "        is automatically resampled if necessary.   If  one  spectrum  is\n",
+    "        longer  than  the  other,  only the smaller number of pixels are\n",
+    "        affected.  To exit this mode type 'q'.\n",
+    "        \n",
+    "The keystrokes listed in the checklist are available  in  the  function  mode.\n",
+    "        Binary  operations  with a constant or a second spectrum produce\n",
+    "        a query for the constant value or spectrum name.\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Unit-related operations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Because `Spectrum1D` objects carry their units and spectral axes, many of the unit operations in SPLOT are implicit by virtue of having units associated with a spectrum. Additionally `glue` natively understands the concept of world coordinates (i.e., the spectral coordinate) and corresponding pixel coordinates.  Hence SPLOT-related functionality of setting wavelengths units or scales is most straightforwardly addressed in a notebook setting:\n",
+    "\n",
+    "1. For example, if a user starts with a spectrum that has only pixels but wants to specify a linear wavelength axis, they can simply create one in a new Spectrum1D:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_spec = specutils.Spectrum1D(flux=specviz.current_spectrum.flux,\n",
+    "                     spectral_axis=np.linspace(4000*u.angstrom,7000*u.angstrom, specviz.current_spectrum.flux.size))\n",
+    "specviz.load_data(new_spec)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And an even simpler operation to convert the flux units:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.load_data(specviz.current_spectrum.new_flux_unit(u.Jy))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To change the *display* of the spectrum using specviz is less trivial, but should be possible - while a glue-style approach is feasible:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.app.get_viewer().state.x_att = specviz.app.data_collection.data[0].components[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A simple method that reproduces the above might be better:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "specviz.use_pixel_axis()\n",
+    "specviz.use_spectral_axis()"
+   ]
```

Comment:
This functionality to change display units is crucial both for the viz tool and notebook commands.  The viz tool could use a dropdown where you can either select or type in the desired units.

Reply:
Agreed, will add a bit more about that in the notebook

Reply:
Actually it was already there I think, but I tried to clarify this a bit more.